README.md

```diff
@@ -1 +1,241 @@
-# To-Do-List-Review
\ No newline at end of file
+<a name="readme-top"></a>
+
+<!--
+HOW TO USE:
+This is an example of how you may give instructions on setting up your project locally.
+
+Modify this file to match your project and remove sections that don't apply.
+
+REQUIRED SECTIONS:
+- Table of Contents
+- About the Project
+  - Built With
+  - Live Demo
+- Getting Started
+- Authors
+- Future Features
+- Contributing
+- Show your support
+- Acknowledgements
+- License
+
+OPTIONAL SECTIONS:
+- FAQ
+
+After you're finished please remove all the comments and instructions!
+-->
```

Comment:
You have a described README file, but I kindly suggest you remove all comments thus your code will be clearer, their goal is just to guide you.

Reply:
Yeah I agree to this too. It's going to make your code look more professional by not allowing unused codes (comments) remain in your codebase. So please kindly do so as well.